src/coord/src/coord.rs

```diff
@@ -1860,7 +1836,8 @@ where
             _ => (),
         }
         let mut dataflow = DataflowDesc::new(name);
-        dataflow.set_as_of(as_of);
+        // let as_of = ;
```

Comment:
oops looks like this one slipped through the cracks? @JLDLaughlin I think its fine to bundle getting rid of this in your release notes pr?